parsons/utilities/sql_helpers.py

```diff
@@ -12,17 +12,3 @@ def redact_credentials(sql):
     sql_censored = re.sub(pattern, "CREDENTIALS REDACTED", sql, flags=re.IGNORECASE)
 
     return sql_censored
-
-
-def get_sql_from_file(sql_file):
```

Comment:
Why are we removing this function?

Reply:
Mostly because, as Jason noted, it seems to just be wrapping f.read. That would mean this function isn't adding value beyond just using the File.reader() method. So, I thought that having a whole new function just for this isn't of much use, perhaps?

Reply:
Makes perfect sense, but I'm hesitant to remove anything that someone else might be depending on. It looks like you removed this removal from the PR, but if you wanted to put it back in (or make a separate PR) we should probably put a deprecating warning on and then remove it later.

Reply:
Not sure why github is doing this, at this point it might make sense to just do a fresh PR (my solution for whenever github is confusing me lol)